main/src/library/File.flix

```diff
@@ -456,7 +456,7 @@ namespace File {
             let javaFile = newFile(f);
             let w = newPrintWriter(javaFile);
 
-            Foldable.foldLeft((_, line) -> println(w, line), (), data);
+            Foldable.foreach((line) -> println(w, line), data);
```

Comment:
Extra parens, we can just do `line -> ...` :)

Reply:
Updated this some other places, too. Looks better without too many parenthesis.